### PR TITLE
refactor: Split cozy-ui imports

### PR DESCRIPTION
**Pull request by drazik**
This makes it easier to override components in customizations. Without
that, the customization would have to also override the `react/index.js`
file from cozy-ui. But doing this would force the customization to
maintain this file so it always reflects cozy-ui's original one.

You can see https://testbankssplitcozyuiimports-banks.cozy.works if you want to check that nothing breaks with these changes.

### CODE REVIEW

File: src/ducks/account/AccountSwitch.jsx [hunk 1]
```diff
@@ -6,15 +6,10 @@ import { connect } from 'react-redux'
...
-import {
-  translate,
-  withBreakpoints,
-  Icon,
-  Media,
-  Bd,
-  Img,
-  Overlay
-} from 'cozy-ui/react'
+import { translate, withBreakpoints } from 'cozy-ui/react'
```
Comment (kosssi) [inline src/ducks/account/AccountSwitch.jsx]: finalement on gagne des lignes ^^

File: src/ducks/balance/components/BalanceAccounts.jsx [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import cx from 'classnames'
-import { translate, Button } from 'cozy-ui/react'
+import { translate } from 'cozy-ui/react'
+import Button from 'cozy-ui/react'
```
Comment (ptbrowne) [inline src/ducks/balance/components/BalanceAccounts.jsx]:
```suggestion
import Button from 'cozy-ui/react/Button'
```

File: src/ducks/transactions/actions/HealthLinkAction.jsx [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import React from 'react'
-import { translate, ButtonAction } from 'cozy-ui/react'
+import { translate } from 'cozy-ui/react'
+import ButtonAction from 'cozy-ui/react'
```
Comment (ptbrowne) [inline src/ducks/transactions/actions/HealthLinkAction.jsx]:
```suggestion
import ButtonAction from 'cozy-ui/react/ButtonAction'
```

File: src/ducks/balance/components/BalanceAccounts.jsx (hunk 2 above)
Comment (drazik) [reply src/ducks/balance/components/BalanceAccounts.jsx]: Hawk eyes :eye:

**Pull request merged**